apps/analytics/views.py

```diff
@@ -141,14 +142,23 @@ def get_submission_count(request, challenge_pk, duration):
 )
 @authentication_classes((ExpiringTokenAuthentication,))
 def get_challenge_phase_submission_count_by_team(
-    request, challenge_pk, challenge_phase_pk
+    request, challenge_pk, challenge_phase_pk_or_slug, version
 ):
     """
     Returns number of submissions done by a participant team in a challenge phase
     """
     challenge = get_challenge_model(challenge_pk)
 
-    challenge_phase = get_challenge_phase_model(challenge_phase_pk)
+    if version == 'v1':
```

Comment:
Hey @Kajol-Kumari Can we change the slug based version to `v2` here and similar other places as it is the newer version of the API.